.AUTHORS.aux

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Is this file a leftover?

Reply:
This is the static part of the AUTHORS file that is not generated from the git log.